### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

**Comment by spxtr** [comment]
Nice, that's pretty fast.

**Comment by krzyzacy** [comment]
it probably should delete the previous comment

**Comment by spxtr** [comment]
Baby steps :)

File: prow/config/config.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/ghodss/yaml"
 )
 
+type ThisIsPublic int
```
Comment (spxtr) [inline prow/config/config.go]: Bad!

**Pull request closed without merging**